### PR TITLE
Readme enhanced

### LINKED ISSUE #131: List of Open Source programs in the Readme file
The Readme file can be enhanced by adding a list of all the Open Source programs in whcih this project has been listed in. Alos there are a lot of grammatical errors in the Readme file that needs correction.

### PR DESCRIPTION
**Pull request by anirudhsai20**
# Description

I have added a list of all the Open Source programs in which this project was listed in and I also corrected a minor grammatical error.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Friday/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
Hey @anirudhsai20 Kindly Add SWoC too, and this Project has been part of that program too...

**Comment by anirudhsai20** [comment]
Sure. I will add SWOC as well.

**Comment by anirudhsai20** [comment]
I have made the necessary changes.

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
Kindly Reduce the Size and Keep them in Same Line and add an anchor tag linking to their official websites

**Comment by anirudhsai20** [comment]
Hi! I have made the required changes. Kindly, check if you are satisfied with them.

File: README.md [hunk 1]
```diff
@@ -32,9 +32,26 @@
   
    <h2 align="center"> Video Preview ⏩ <a href="https://youtu.be/IJCo80Y0wjI">  <img src="https://img.shields.io/badge/Click Me%20-%23FF0000.svg?&style=for-the-badge&logo=YouTube&logoColor=white"/> </a>
   
- ## Contributors ✨
+ ## Friday is a part of these Open Source programs
 
-Thanks goes to these wonderful people.
+<p align="center">
+ <a>
+ <img  width="300" height="70" src="https://raw.githubusercontent.com/GirlScriptSummerOfCode/MentorshipProgram/master/GSsoc%20Type%20Logo%20Black.png">
```
Comment (avinashkranjan) [inline README.md:39]: `[](https://gssoc.girlscript.tech/)`

Change this code to this to link the image to the website

File: README.md [hunk 2]
```diff
@@ -32,9 +32,26 @@
...
+
+ <img  width="100" height="100" src="https://swoc.tech/img/logo-2.png">
```
Comment (avinashkranjan) [inline README.md:41]: Follow the Same here

**Review by avinashkranjan — CHANGES_REQUESTED** [review]
Make the asked changes

**Comment by anirudhsai20** [comment]
Hi! Sorry I made a lot of mistakes. I have corrected them now. Please check if they require any other changes.

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**